recipe/recipe.yaml

```diff
@@ -122,5 +123,4 @@ about:
 
 extra:
   recipe-maintainers:
-    - erip
```

Comment:
Why is this maintainer being removed?

Reply:
https://github.com/conda-forge/triton-feedstock/pull/38/commits/f725c1e7cfebd820a6d0731186a4f7894faced16